addon/components/frost-object-browser.js

```diff
@@ -22,6 +22,9 @@ export default Ember.Component.extend({
   classNames: ['frost-object-browser'],
   contentHeight: 505,
   detailLevel: 'low',
+  isHighDetailVisible: true,
+  isLowDetailVisible: true,
+  isMediumDetailVisible: false,
```

Comment:
There is a UX change to no longer show medium but I figured why not leave it in the open source component as optional. I thought making each level configurable would be nice. For example if I am a consumer and my medium level of details aren't rendering properly I can just toggle a flag until I fix it.